src/main/java/com/example/onlinebookshop/entities/Book.java

```diff
@@ -5,24 +5,39 @@
 import jakarta.persistence.GeneratedValue;
 import jakarta.persistence.GenerationType;
 import jakarta.persistence.Id;
+import jakarta.persistence.Table;
 import java.math.BigDecimal;
 import lombok.Data;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.SQLRestriction;
 
 @Data
```

Comment:
dont use data with entity as it cause bugs, replace with getter/setter annotations